assessment_software/src/app/course-card/course-card.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="course-box">
+    <button (click)="handleClick($event)"><svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M19 3V3C19.1857 3 19.2786 3 19.3569 3.00685C20.2271 3.08298 20.917 3.77292 20.9932 4.64312C21 4.72141 21 4.81427 21 5V16.12C21 17.1281 21 17.6321 20.8038 18.0172C20.6312 18.3559 20.3559 18.6312 20.0172 18.8038C19.6321 19 19.1281 19 18.12 19H5.88C4.87191 19 4.36786 19 3.98282 18.8038C3.64413 18.6312 3.36876 18.3559 3.19619 18.0172C3 17.6321 3 17.1281 3 16.12V5C3 4.81427 3 4.72141 3.00685 4.64312C3.08298 3.77292 3.77292 3.08298 4.64312 3.00685C4.72141 3 4.81427 3 5 3V3" stroke="#33363F" stroke-width="2"/><path d="M19 3H18.118C17.6329 3 17.3903 3 17.185 3.09862C17.1214 3.12917 17.0612 3.16636 17.0055 3.20958C16.8255 3.34909 16.717 3.56606 16.5 4V4C16.283 4.43394 16.1745 4.65091 15.9945 4.79042C15.9388 4.83364 15.8786 4.87083 15.815 4.90138C15.6097 5 15.3671 5 14.882 5H8.91667C8.51499 5 8.31415 5 8.14071 4.93323C8.00297 4.8802 7.87867 4.79733 7.77675 4.69058C7.6484 4.55617 7.57116 4.37078 7.41667 4V4C7.26218 3.62922 7.18493 3.44383 7.05659 3.30942C6.95466 3.20267 6.83036 3.1198 6.69262 3.06677C6.51918 3 6.31834 3 5.91667 3H5" stroke="#33363F" stroke-width="2"/><path d="M21 8H18.118C17.6329 8 17.3903 8 17.185 8.09862C17.1214 8.12917 17.0612 8.16636 17.0055 8.20958C16.8255 8.34909 16.717 8.56606 16.5 9V9C16.283 9.43394 16.1745 9.65091 15.9945 9.79042C15.9388 9.83364 15.8786 9.87083 15.815 9.90138C15.6097 10 15.3671 10 14.882 10H8.91667C8.51499 10 8.31415 10 8.14071 9.93323C8.00297 9.8802 7.87867 9.79733 7.77675 9.69058C7.6484 9.55617 7.57116 9.37078 7.41667 9V9C7.26218 8.62922 7.18493 8.44383 7.05659 8.30942C6.95466 8.20267 6.83036 8.1198 6.69262 8.06677C6.51918 8 6.31834 8 5.91667 8H3" stroke="#33363F" stroke-width="2"/><path d="M21 13H18.118C17.6329 13 17.3903 13 17.185 13.0986C17.1214 13.1292 17.0612 13.1664 17.0055 13.2096C16.8255 13.3491 16.717 13.5661 16.5 14V14C16.283 14.4339 16.1745 14.6509 15.9945 14.7904C15.9388 14.8336 15.8786 14.8708 15.815 14.9014C15.6097 15 15.3671 15 14.882 15H8.91667C8.51499 15 8.31415 15 8.14071 14.9332C8.00297 14.8802 7.87867 14.7973 7.77675 14.6906C7.6484 14.5562 7.57116 14.3708 7.41667 14V14C7.26218 13.6292 7.18493 13.4438 7.05659 13.3094C6.95466 13.2027 6.83036 13.1198 6.69262 13.0668C6.51918 13 6.31834 13 5.91667 13H3" stroke="#33363F" stroke-width="2"/></svg></button>
+    <p style="font-size: 24px;">{{courseName}}</p>
```

Comment:
The course name and the archive button should be on the same level. currently the archive button is on top of the course name. 

it is better to wrap them with a div so that they are on the same level:


```
<div>
   <app-button></app-button>
   <p>{course_name}<p>
<div>
```